build.sh

```diff
@@ -53,6 +53,21 @@ git config --global user.name "$commit_username"
 git config --global user.email "$commit_email"
 echo '::endgroup::'
 
+echo '::group::Updating pkgver'
+if grep -q 'pkgver()' $pkgbuild; then
```

Comment:
`grep` is error-prone. What if `pkgver()` only appears in irrelevant places (such as inside `pkgdesc` string)? What if it was `pkgver ()` instead?

I think that instead of trying to be smart, it is better that we let users determine it themselves (by inputting a `with` value).